packages/block-library/src/heading/deprecated.js

```diff
@@ -48,7 +52,52 @@ const migrateCustomColors = ( attributes ) => {
 	};
 };
 
+const migrateTextAlign = ( attributes ) => {
+	const { align, ...rest } = attributes;
+	return TEXT_ALIGN_OPTIONS.includes( align )
+		? { ...rest, textAlign: align }
+		: attributes;
+};
+
+const TEXT_ALIGN_OPTIONS = [ 'left', 'right', 'center' ];
```

Comment:
It might be easier to follow code if this const was defined before it is used.

Reply:
You're definitely right.